ios/FluentUI/Resources/Localization/en.lproj/Localizable.stringsdict

```diff
@@ -2,6 +2,28 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>Accessibility.AvatarGroup.Overflow.Value</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@others@</string>
+		<key>others</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>d</string>
+			<key>one</key>
+			<string>%d other</string>
+			<key>two</key>
```

Comment:
What is the reason for needing `few`, `many`, and `other` all as keys?

I assume they are probably all required?

Reply:
Everything but `other` is optional, but it gives the other languages more options.